frontend/src/locales/en-US.json

```diff
@@ -878,7 +878,8 @@
     "not-found-last-successful-sync-was": "Not found, last successful sync was {time}",
     "view-unassigned-databases": "View unassigned databases",
     "unassigned-databases": "Unassigned databases",
-    "restore-database": "Restore database"
+    "restore-database": "Restore database",
+    "selected-n-databases": "Selected {n} database | Selected {n} databases"
```

Comment:
```suggestion
    "selected-n-databases": "{n} selected"
```
So we can get away with database(s).

Reply:
And this sounds more generic. We can also use "n selected" for other selection scenarios.

Reply:
FYI @candy2255 wdyt?

Reply:
{n} database selected | {n} databases selected